cmd/mps-control-daemon/mps/daemon.go

```diff
@@ -124,25 +127,51 @@ func (d *Daemon) Start() error {
 	}
 	defer statusFile.Close()
 
+	klog.InfoS("Starting log tailer", "resource", d.rm.Resource())
+	tail := exec.Command("tail", "-n", "+1", "-f", filepath.Join(logDir, "control.log")) //nolint:gosec
+	tail.Stdout = os.Stdout
+	if err := tail.Start(); err != nil {
+		klog.ErrorS(err, "Could not start tail command on control.log; ignoring logs")
+		d.tail = nil
+	} else {
+		d.tail = tail
+	}
+
 	return nil
 }
 
 // Stop ensures that the MPS daemon is quit.
 func (d *Daemon) Stop() error {
-	output, err := d.EchoPipeToControl("quit")
+	_, err := d.EchoPipeToControl("quit")
 	if err != nil {
 		return fmt.Errorf("error sending quit message: %w", err)
 	}
-	klog.InfoS("Shut down MPS", "output", output)
+	klog.InfoS("Stopped MPS control daemon", "resource", d.rm.Resource())
```

Comment:
Do these klog outputs get mixed with the tail log?

Reply:
Yes. klog prints to stderr by default and that's captured by the container logger.